utilities/_utilities.responsive-spacings.scss

```diff
@@ -9,8 +9,7 @@
  * maps, e.g.:
  *
  *   $inuit-responsive-spacing-directions: (
- *     null: null,
- *     bottom: bottom,
+ *     "-bottom": "-bottom",
```

Comment:
Why did you remove `null`? Further down in the CSS comment block, it says:

Reply:
Correct, null creates the versions without `$inuit-responsive-spacing-sizes`.
Fixed your comment